maintainers/scripts/travis-nox-review-pr.sh

```diff
@@ -27,7 +27,11 @@ elif [[ $1 == build ]]; then
     nix-build nixos/release.nix -A options
 
     echo "=== Checking tarball creation"
-    nix-build pkgs/top-level/release.nix -A tarball
+    if [[ $TRAVIS_OS_NAME == "osx" ]]; then
+        echo "Skipped, as not working on darwin"
```

Comment:
`error: a ‘x86_64-linux’ is required to build ‘/nix/store/0illn11rkdw7v4cfparg2z0yfds9qdli-nixpkgs-tarball-16.09pre1234.abcdef.drv’, but I am a ‘x86_64-darwin’`

Check out this build: https://travis-ci.org/mpscholten/nixpkgs/jobs/134120897 It's a build without skipping the tarball.